src/codelabs/implicit-animations.md

```diff
@@ -191,7 +191,7 @@ to set `opacity` to 1:
 ```
 
 {{site.alert.secondary}}
-  Notice that you only need to set the start and end values of `opacity`.
+  Only set the start and end values of `opacity`.
```

Comment:
I think the previous wording here was important, since it's not telling them to only set start and end, but explaining why they don't have to worry about the in-between.